src/screens/Topics/index.js

```diff
@@ -38,6 +40,7 @@ const Topics = () => {
   const [, dispatch] = React.useContext(Context).topics;
   const [myTopic, setMyTopic] = React.useState({});
   const [isPreload, setIspreload] = React.useState(true);
+  const {bottom} = useSafeAreaInsets();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useSafeAreaInsets` hook to create a new state variable `bottom` is a good practice. This will help to dynamically calculate the marginBottom for the scrollViewStyle and recyclerview style property, improving the visual consistency of the components.
```diff
+ const {bottom} = useSafeAreaInsets();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->